api/src/main/java/org/hawkular/apm/api/services/AbstractConfigurationService.java

```diff
@@ -59,20 +57,6 @@
 
                 if (handler != null) {
                     handler.init(processor);
-
-                    if (handler.getIssues() != null) {
-                        for (Issue issue : handler.getIssues()) {
```

Comment:
This part is still required - we need a way for the handler.init method to be able to record issues somewhere so that they can be returned as ConfigMessages from the REST serivce to the UI which will display the errors. So it doesn't need to be done using ProcessorIssue or Issue class - but some way (possibly even ConfigMessage itself) to get the errors from the init method.